### PR TITLE
Updated Community Section

### PR DESCRIPTION
**Pull request by jasmineanand**
**Summary:**
I made updates to the community page, including the addition of forms for donations, story submissions, and job opportunities. I also added a subscription button at the end of the page and created three inspiring stories. In the Work and Volunteer Opportunities section, I added three cards for Freelancing, Web Development, and Digital Marketing, where users can access YouTube tutorials by clicking "Learn," as discussed with Sristy.

**Changes Made:**
Implemented forms for donations, writing stories, and job opportunities.
Added a subscription button at the end of the page.
Created three inspiring stories.
Developed three skill cards under Work and Volunteer Opportunities for Freelancing, Web Development, and Digital Marketing, each linking to relevant YouTube tutorials.

**Testing Performed:**
While testing, I noted that clicking the buttons, such as submitting the forms, does not currently redirect anywhere. I am still working on this functionality and plan to update it so that the form details are forwarded to an email or another designated endpoint.

**Related Issues:**
I'm encountering difficulties with the dark/light mode toggle on the community page—it's not functioning as expected. It would be helpful if Sristy could take a look. Additionally, I need clarification on the Work and Volunteer Opportunities section. Should I delete this section, or are there specific changes you’d like me to make?

**Additional Notes:**
Please share your feedback and let me know if there are any further changes I should implement.

### CODE REVIEW

**Review by sristy17 — CHANGES_REQUESTED** [review]
[Screencast from 2024-08-22 23-19-22.webm](https://github.com/user-attachments/assets/be86bda4-5dfe-4ed1-ae90-2ea3057e4c7b)


Please fix this issue.

File: views/pages/community.ejs
Comment (sristy17) [inline views/pages/community.ejs:7]: Change this to style_community.css

**Review by sristy17 — CHANGES_REQUESTED** [review]

File: views/pages/community.ejs
Comment (jasmineanand) [reply views/pages/community.ejs]: yeah done

**Comment by jasmineanand** [comment]
Hi @sristy17 , could you please review and approve this PR when you get a chance? We need your approval to proceed with the merge. Thanks!

**Pull request merged**
